MiniAn/minian_main.py

```diff
@@ -555,28 +571,28 @@ def save_minian_to_doric(
 
         print(mn_defs.Messages.SAVE_ROI_SIG, flush=True)
         pathROIs = vpath+mn_defs.DoricFile.Group.ROISIGNALS+operationCount+'/'
-        utils.save_roi_signals(C.values, A.values, time_, f, pathROIs+vdataset, attrs_add={"RangeMin": 0, "RangeMax": 0, "Unit": "AU"})
```

Comment:
I think `time` is reserved for something else in python, so I would be careful to remove that `_`

Reply:
Ok I will check that